packages/xml-odata-annotation-converter/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.json"
+  ],
+  "exclude": [
+    "dist",
+    "node_modules"
+  ],
+  "compilerOptions": {
```

Comment:
Please check the options in the root `tsconfig.json` and remove the duplicate ones.